SETUP.md

```diff
@@ -269,3 +269,29 @@ Additionally, you must install the [spark-cosmosdb connector](https://docs.datab
    7. Restart the cluster.
 
 </details>
+
+## Setup guide for Docker
+
+Docker images of the repo for different environment are provided. Dockerfiles can be found at `Recommenders/reco_utils/docker` if one would like to build Docker images on his/her own.
```

Comment:
suggested wording change and link addition (if we land on a link, maybe leave it out till then):

Docker images of the repo for different environments are available (provide link). To build Docker images yourself use the Dockerfiles in `reco_utils/docker`